config-user.default.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+# Do not modify this file, which defines default values, but create a
+# `./data/config-user.custom.php` file instead, containing the keys you want to
+# override.
```

Comment:
Footnote: GitHub views are fine up to 131 columns ;-)

Reply:
It's just that I'm using `gqq` in Vim which automatically wrap my lines :) But I also try to limit the lines to 80/100 columns so it looks good when I split my screen